bootstrap.d/dev-libs.yml

```diff
@@ -162,6 +162,28 @@ packages:
           DESTDIR: '@THIS_COLLECT_DIR@'
         quiet: true
 
+  - name: json-c
+    source:
+      subdir: ports
+      git: 'https://github.com/json-c/json-c.git'
+      branch: 'master'
+      tag: 'json-c-0.15-20200726'
```

Comment:
Still missing version information, `version: 0.15` just below the `tag` line should do the trick